### DISCUSSION TITLE
Sync with Goodnotes?

### DISCUSSION THREAD

**Discussion by edogab33** [Future-Scholars/paperlib#496] [General]
I usually save papers from my laptop and read them on Goodnotes from my tablet because I can highlight phrases, write notes, etc. Is there any way to sync a shared folder between these two apps?

**Comment by GeoffreyChen777** [comment]
Yes, but with cloud storage services such as Dropbox, and Onedrive.

1. Use a folder in these cloud storage as your library folder like this:

<img width="930" alt="image" src="https://github.com/Future-Scholars/paperlib/assets/14183213/a8d9b986-38f7-47e3-a450-b38132adf4c2">

2. Now all your PDFs should be synced by these cloud services.
3. Install Dropbox/Onedrive etc. on your iPad.
4. Use goodnotes to open files in the cloud folder.


Some tips:

1. Custom the renaming format:

<img width="749" alt="image" src="https://github.com/Future-Scholars/paperlib/assets/14183213/640f3f50-bac7-45bc-8a5b-32960913d021">

You can get a neat folder system structure like:

```
| - 2022
| - 2023
     | - Journal A
     | - Journal B
         | - Paper A.pdf
         | - Paper B.pdf
```

2. If you wish to sync metadata between different PCs, please follow [this](https://paperlib.app/en/doc/cloud-sync/setup.html). Please DON'T use Dropbox etc. to sync database files.

**Reply by edogab33** [reply]
Thanks a lot!
